db/seeds.rb

```diff
@@ -1,7 +1,36 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Article.destroy_all
+User.destroy_all
+journalist = User.create(email: "journalist@mail.com", role: "journalist", password: "password")
+registered = User.create(email: "registered@mail.com", role: "registered", password: "password")
+articles = Article.create(
+  [
```

Comment:
You don't have to wrap them in variables like `articles =` if you're not going to use the variables for anything